src/VuetifyResource.vue

```diff
@@ -782,6 +782,13 @@
                 }
                 return canUpdate;
             },
+
+            cancelUpdate(){
+                //Setting route without hash
+                this.$router.push({ name: this.$route.name, params:  this.$route.params });
```

Comment:
this is a dangerous solution, you dont know if the user has vue-router, or if params and name are the only router options they use. I would use native javascript code just like the rest of this plugin does for the hash

`window.location.hash = ''`